src/plugins/kibana_usage_collection/server/collectors/management/telemetry_management_collector.ts

```diff
@@ -19,26 +19,115 @@
 
 import { IUiSettingsClient } from 'kibana/server';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { includes } from 'lodash';
 import { stackManagementSchema } from './schema';
+import { WhitelistedSettings, BlacklistedSettings } from './types';
 
-export interface UsageStats extends Record<string, boolean | number | string> {
-  // We don't support `type` yet. Only interfaces. So I added at least 1 known key to the generic
-  // Record extension to avoid eslint reverting it back to a `type`
-  'visualize:enableLabs': boolean;
-}
+export type UsageStats = WhitelistedSettings | BlacklistedSettings;
+
+const whitelistedSettings: Array<keyof WhitelistedSettings> = [
```

Comment:
AFAIRecall, we discussed the possibility of adding a key in the UI Setting's registration API to exclude/mask the registered setting. Do we have any follow-up issue to replace this list with that API modification?